eng/Version.Details.xml

```diff
@@ -232,11 +236,7 @@
       <Sha>93a261562a815434ceaae0e3c5c15fb6e457f9c4</Sha>
       <SourceBuild RepoName="source-build-reference-packages" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.XliffTasks" Version="1.0.0-beta.23523.1" CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk">
-      <Uri>https://github.com/dotnet/xliff-tasks</Uri>
-      <Sha>83531fff1a39abe38de53e2e6b702bc1bb9c3bb4</Sha>
-      <SourceBuild RepoName="xliff-tasks" ManagedOnly="true" />
-    </Dependency>
+
```

Comment:
Is this blank line intentional?  I ask because I don't usually see blank lines in Verion.Details.xml files